.github/workflows/release.yml

```diff
@@ -78,11 +84,20 @@ jobs:
           path: /home/runner/work/${{ env.GITHUB_REPO_NAME }}/${{ env.GITHUB_REPO_NAME }}/build/reports/tests/test
       - name: 'Create release'
         run: ./gradlew createRelease -Prelease.forceVersion=${{github.event.inputs.force_version}}
+      - name: 'Get version'
+        id: get-version
+        run: |
+          VERSION=$(./gradlew cV -q -Prelease.quiet)
+          echo "Pushing version v$VERSION to ${{ inputs.aws_region }}"
```

Comment:
This action is used for the components as well and they are missing `inputs.aws_region`

Reply:
Changed the wording to not contain `inputs.aws_region`, thanks.